src/laboratory/templates/laboratory/shelfobject/action_modal.html

```diff
@@ -9,11 +9,17 @@
 {% trans 'Add' as increase_so_tittle %}
 {% include 'modal_template.html' with add_shelf_info=True shelf_info_position="top" form=increase_object_form id="increasesomodal" title=increase_so_tittle form_id="increasesoform" url=increase_so_url %}
 
+{% url "laboratory:api-shelfobject-fill-increase-shelfobject" lab_pk=laboratory org_pk=org_pk as increase_ma_url %}
+{% trans 'Add' as increase_ma_tittle %}
+{% include 'modal_template.html' with add_shelf_info=True shelf_info_position="top" form=increase_material_object_form id="increasemamodal" title=increase_ma_tittle form_id="increasemaform" url=increase_ma_url %}
 
 {% url "laboratory:api-shelfobject-fill-decrease-shelfobject" lab_pk=laboratory org_pk=org_pk as decrease_so_url %}
 {% trans 'Substract' as decrease_so_tittle %}
 {% include 'modal_template.html' with add_shelf_info=True shelf_info_position="top" form=decrease_object_form id="decreasesomodal" title=decrease_so_tittle form_id="decreasesoform" url=decrease_so_url %}
 
+{% url "laboratory:api-shelfobject-fill-decrease-shelfobject" lab_pk=laboratory org_pk=org_pk as decrease_ma_url %}
+{% trans 'Substract' as decrease_ma_tittle %}
```

Comment:
It is not neccessary include url and title variables, because these varibales already defined.